gateleen-core/src/main/java/org/swisspush/gateleen/core/storage/HttpResourceStorage.java

```diff
@@ -86,12 +89,14 @@ public int getPort() {
     public void put(final String uri, MultiMap headers, Buffer buffer, final Handler<Integer> doneHandler) {
         client.request(HttpMethod.PUT, uri).onComplete(asyncResult -> {
             if (asyncResult.failed()) {
-                log.warn("Failed request to {}: {}", uri, asyncResult.cause());
+                log.warn("Failed request to {}", uri, new Exception("stacktrace", asyncResult.cause()));
+                log.error/*TODO*/("TODO why do we not report this error to our caller?!? Is this maybe"
+                        + " the explanation for all our unexplainable timeouts or lost messages we"
```

Comment:
We should not add logs for personal bug hunting to the codebase. Just build your own release locally and add the logs there